### PR TITLE
Helm Release wordpress Typescript example

### PR DESCRIPTION
**Pull request by viveklak**

### CODE REVIEW

File: kubernetes-ts-helm-release-wordpress/index.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import * as k8s from "@pulumi/kubernetes";
+
+// Deploy the bitnami/wordpress chart.
+const wordpress = new k8s.helm.v3.Release("wpdev", {
+    chart: "wordpress",
+    repositoryOpts: {
+        repo: "https://charts.bitnami.com/bitnami",
+    },
+    version: "13.0.6",
+    // Remove these values if your cluster can
```
Comment (lblackstone) [inline kubernetes-ts-helm-release-wordpress/index.ts:13]: Looks like this comment got cut off

File: kubernetes-ts-helm-release-wordpress/README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+ +   pulumi:pulumi:Stack               ts-helm-release-wordpress-dev  create     
+ +   └─ kubernetes:helm.sh/v3:Release  wpdev                          create     
+ 
+Resources:
+    + 2 to create
+
+Do you want to perform this update? yes
+Updating (dev)
+
+View Live: https://app.pulumi.com/vivek/ts-helm-release-wordpress/dev/updates/1
```
Comment (lblackstone) [inline kubernetes-ts-helm-release-wordpress/README.md:48]:
```suggestion
View Live: https://app.pulumi.com/<redacted>/ts-helm-release-wordpress/dev/updates/1
```

File: kubernetes-ts-helm-release-wordpress/package.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "wordpress",
+  "devDependencies": {
+    "@types/node": "^12.0.0"
+  },
+  "dependencies": {
+    "@pulumi/kubernetes": "^3.0.0",
```
Comment (lblackstone) [inline kubernetes-ts-helm-release-wordpress/package.json:7]: Probably need to pin this to a recent dep for this to work.

**Review by lblackstone — APPROVED** [review]
Few minor comments, but looks good overall

File: kubernetes-ts-helm-release-wordpress/index.ts (hunk 1 above)
Comment (viveklak) [reply kubernetes-ts-helm-release-wordpress/index.ts]: Fixed.

File: kubernetes-ts-helm-release-wordpress/README.md (hunk 2 above)
Comment (viveklak) [reply kubernetes-ts-helm-release-wordpress/README.md]: Fixed.

File: kubernetes-ts-helm-release-wordpress/package.json (hunk 3 above)
Comment (viveklak) [reply kubernetes-ts-helm-release-wordpress/package.json]: Bumped to 3.15.0

**Pull request merged**